textdescriptives/components/pos_stats.py

```diff
@@ -20,23 +20,26 @@ class POSStatistics:
     """spaCy v.3.0 component that adds attributes for POS statistics to `Doc` and `Span` objects.
     """
 
-    def __init__(self, nlp: Language):
+    def __init__(self, nlp: Language): # Is the parameter-hint incorrect, should it be "model" instead?
```

Comment:
spacy uses `Language` in their repositories, so let's use that to keep it consistent